src/modules/AdminModule.cpp

```diff
@@ -34,10 +31,6 @@
 #include "modules/PositionModule.h"
 #endif
 
-#if !defined(ARCH_PORTDUINO) && !defined(ARCH_STM32WL) && !MESHTASTIC_EXCLUDE_ENVIRONMENTAL_SENSOR
-#include "AccelerometerThread.h"
-#endif
-
```

Comment:
Is this just some leftover cleanup like the main.h above or was this required for actual functionality?